lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3376,6 +3391,22 @@ namespace {
                                 std::nullopt);
     }
 
+    bool cxxMethodCanBeRefcountConventionAnnotated(const clang::CXXMethodDecl *method) {
```

Comment:
Can we make the name slightly more brief, e.g. `canBeRefCountConventionAnnotated`?

Reply:
I agree. The `cxxMethod` part can be inferred from the signature. I think the goal of these names to convey the info that is not clear from the types.